py3dtilers/GeojsonTiler/README.md

```diff
@@ -20,12 +20,21 @@ geojson-tiler --paths ../../geojson/
 ```
 It will read all .geojson and .json it the _geojson_ directory and parse them into 3DTiles. It will also create a single .obj model from all readed files.
 
+### LOA
+Using the LOA\* option creates a tileset with a __refinement hierarchy__. The leafs of the created tree are the detailled features (features loaded from the data source) and their parents are LOA geometries of those detailled features. The LOAs are 3D extrusions of the polygons defined by roads. The polygons must be given as a path to a directory containing geojson file(s) (the features in those geojsons must be Polygons or MultiPolygons).
```

Comment:
Maybe we should avoid the bias (implicit limitation) relating LOA as being bounded/constructed-out-of roads ? Maybe we could think of alternative strategies for constructing/using LOAs (like boroughs, rivers or other geographical partition)...